### DISCUSSION TITLE
Welcome to go-zero Discussions!

### DISCUSSION THREAD

**Discussion by kevwan** [zeromicro/go-zero#257] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

  BTW: don't forget to **star** this project if you like it 💪.

**Comment by xzq0** [comment]
very good

**Comment by dmitryburov** [comment]
It's a great idea! TNX

**Reply by kevwan** [reply]
> It's a great idea! TNX

Thanks for investigating go-zero!

**Comment by xiaohan1105** [comment]
赞个
